### ISSUE TITLE
How to suggest features?

### ISSUE THREAD

**Issue by adex720** [adex720/Minigames#1]
To leave your suggestion create a new issue.
Before starting your suggestion make sure to check these things:
- Check that no issue with the same suggestion already exists.
- Don't suggest jokes.
After that you can create a new issue to suggest it. Make sure that it follows these rules:
- Make a short but clear title for the suggestion.
- Fully explain the suggestion. Tell how it functions and what prizes it has. Everyone who knows the basics of the bot should understand how it works.
- Add the __enchantment__ label to the issue. If it's a new minigame also add the __new minigame__ label.
- If you want to code it yourself mention it but don't create a pr before it's been confirmed that it will be added.
After leaving your suggestion:
- Not getting a reply means nothing. Everything will get read.
- Do not ask people to come "see" it.
- Understand that not everything will be added.
- Understand that the idea might be modified.
